### DISCUSSION TITLE
Interpolation in @html_str

### DISCUSSION THREAD

**Discussion by lassepe** [fonsp/Pluto.jl#1109] [Questions about using Pluto] [answered]
Is there a way to interpolate a variable in to a `html""` string? Specifically, I'm trying to do something like this:

```julia
slide_width = 800

html"""
<style>
main {
    max-width: $(slide_width)px;
}
"""
```
However, the `$` appears in the content of the html string. Do I need to escape it somehow?

**Comment by fonsp** [comment, marked as answer]
We have https://github.com/clarkevans/HypertextLiteral.jl for this!
